utils/tests/test_utils.py

```diff
@@ -1,10 +1,23 @@
 from unittest import TestCase
-
-from utils.network import is_udp_port_available
+from utils.utils import cores_for_worker_process
 
 
 class TestNetwork(TestCase):
-    def test_udp(self):
-        is_udp_port_available(50301)
+    def test_cpu_affinity(self):
+        num_workers = 44
```

Comment:
This does not make sense. Why would you remove the is_udp_port_available line and add the rest of the code instead?
I think everything was okay and you shouldn't have changed it

Reply:
The cores_for_worker_process method was in the utils folder and hence I shifted it here